### PR TITLE
PUD-1059 Add Welsh recall info leaflet to Dossier for Welsh LDUs

### PR DESCRIPTION
**Pull request by AndyNewton**
PUD-1059 Add Welsh recall info leaflet to Dossier for Welsh LDUs

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/dossier/DossierContextFactory.kt [hunk 1]
```diff
@@ -43,6 +43,10 @@ data class DossierContext(
       currentPrisonName,
       recall.bookingNumber!!
     )
+
+  fun includeWelsh(): Boolean {
```
Comment (philsegal) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/dossier/DossierContextFactory.kt:47]: not sure if that's the best name, include suggests there could be others? 'welshOnly'?

Comment (philsegal) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/dossier/DossierContextFactory.kt:47]: ah, I see now. `lduIsInWales`?

**Review by philsegal — APPROVED** [review]
Small naming comment, else looks good.

File: src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/dossier/DossierContextFactory.kt (hunk 1 above)
Comment (AndyNewton) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/dossier/DossierContextFactory.kt:47]: I dunno what would really be better.  Everything else is already included.  This is the line of usage:
`createTableOfContentsDocumentMap(reasonsForRecallPdfBytes, recallId, dossierContext.includeWelsh())`
and the result is to add the Welsh doc.  🤷‍♂️

**Pull request merged**